challenge.js

```diff
@@ -42,7 +42,7 @@
  */
 
 // ANSWER
-
+A.child[3];
```

Comment:
This question was looking for a CSS selector string. Here is an explanation and sample solution:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

However, this resembles something we can do if we have a reference to A's DOM node in JS stored in the variable `A`. Every DOM node has a `children` property that contains an array of references to its child elements, in order. `A.children[2]` would retrieve D for you, which you can then use to retrieve E: `A.children[2].children[0]`.

1/2